README.md

```diff
@@ -638,3 +639,37 @@ One of three out comes can occur after the export retried has completed.
 
 > *NOTE: Changing Writable.Pipeline.ExecutionOrder will invalidate all currently stored data and result in it all being removed from the database on the next retry.* This is because the position of the export function can no longer be guaranteed and no way to ensure it is properly executed on the retry.
 
+### Secrets
+
+#### Getting Secrets
+
+Application Services can retrieve their secrets from the underlying secret store using the [GetSecrets()](#.GetSecrets()) API in the SDK. 
+
+##### Vault Secrets
+
+If in secure mode, the secrets are retrieved from Vault based on the SecretStore configuration values. If tokenfile is present, it will override the Authentication.AuthToken value. The path parameter in the GetSecrets() API call is appended to the SecretStore.Path configuration and used as the secret's location in Vault. 
```

Comment:
`If tokenfile is present, it will override the Authentication.AuthToken value` seems out of place. I.e. belongs in a section on configuring SecretStore, which we don't have and is beyond this PR. Probably should add an issue for this. For now just remove.